gravitee-am-service/src/main/java/io/gravitee/am/service/model/NewReporter.java

```diff
@@ -33,6 +33,9 @@
     @NotNull
     private String name;
 
+    @NotNull
+    private boolean system;
```

Comment:
As this object is used to create a reporter from the REST resource, we should reject a reporter creation if the system boolean is true in this method https://github.com/gravitee-io/gravitee-access-management/blob/2c9268c79b42d2ff050c1a7ea3ec284002d42cd1/gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/ReportersResource.java#L106